compiler+runtime/include/cpp/jank/runtime/obj/detail/base_persistent_map.hpp

```diff
@@ -46,16 +46,32 @@ namespace jank::runtime::obj::detail
 
     static void to_string_impl(typename V::const_iterator const &begin,
                                typename V::const_iterator const &end,
-                               fmt::memory_buffer &buff)
+                               fmt::memory_buffer &buff,
+                               native_bool to_code = false)
```

Comment:
Please make this `const`.

Reply:
Also, please don't use default arguments. We can update all of the calls to this. As soon as there's one default arg, adding any other args requires them to be default args, too.